packages/contracts/test/LendingPool.t.sol

```diff
@@ -0,0 +1,394 @@
+    AToken public aTokenImpl;
+    StableDebtToken public stabledebtTokenImpl;
+    VariableDebtToken public variabledebtTokenImpl;
+
+    // system contracts
+    LendingPoolAddressesProvider public lpAddressProvider;
+    LendingPoolConfigurator public lpConfigurator;
+    LendingPoolConfigurator public proxyConfigurator;
+    DefaultReserveInterestRateStrategy public strategy;
+    LendingRateOracle public oracle;
+    LendingPool public implementationLp;
+    LendingPool public proxyLp ;
+    ProxyAdmin public proxyAdmin;
+
+    function setUp() public {
+        vm.label(admin, 'admin');
+        vm.label(owner, 'owner');
+        vm.label(alice, 'alice');
+
+        _configureOpMainnet();
+        _configureBase();
+
+    }
+
+    function _configureOpMainnet() internal {
+        opMainnet = vm.createSelectFork("https://mainnet.optimism.io/");
+        
+        // Underlying
+        Underlying = new MockERC20("Mock rupee","Underlying"); 
+        vm.label(address(Underlying), "Underlying");
+        Underlying.mint(alice, 1000_000);
+
+        // implementation aToken
+        aTokenImpl = new AToken();
+        
+        // implementation stabledebtToken
+        stabledebtTokenImpl = new StableDebtToken();
+        
+        // implementation variabledebtToekn
+        variabledebtTokenImpl = new VariableDebtToken();
+
+        // lendingPoolAddressProvider
+        lpAddressProvider = new LendingPoolAddressesProvider("Underlying",owner,owner);
+        vm.label(address(lpAddressProvider), "lpAddressProvider");
+        
+        // superchainAsset for opMainnet
+        superchainAsset = new SuperchainAsset("superchainAsset","SCA",18,address(Underlying),ILendingPoolAddressesProvider(address(lpAddressProvider)),owner);
+        vm.label(address(superchainAsset), "superchainAsset");
+
+        // implementation LendingPool
+        implementationLp = new LendingPool();
+        vm.label(address(implementationLp), "implementationLp");
+
+        // admin
+        vm.prank(owner);
+        proxyAdmin = new ProxyAdmin();
+        admin = address(proxyAdmin);
+        
+        // proxy LendingPool
+        vm.prank(owner);
+        lpAddressProvider.setLendingPoolImpl(address(implementationLp));
+        proxyLp = LendingPool(lpAddressProvider.getLendingPool());
+        vm.label(address(proxyLp), "proxyLp");
+
+        // settings in addressProvider
+        vm.prank(owner);
+        lpAddressProvider.setPoolAdmin(admin);
+
+        // implementation configurator
+        lpConfigurator = new LendingPoolConfigurator();
+
+        // proxy configurator
+        vm.prank(owner);
+        lpAddressProvider.setLendingPoolConfiguratorImpl(address(lpConfigurator));
+        proxyConfigurator = LendingPoolConfigurator(lpAddressProvider.getLendingPoolConfigurator());
+
+        // strategy
+        strategy = new DefaultReserveInterestRateStrategy(ILendingPoolAddressesProvider(address(lpAddressProvider)),1,2,3,4,5,6);
```

Comment:
this should be ray terms

Reply:
check this file out https://github.com/aave/protocol-v2/blob/master/markets/aave/rateStrategies.ts